### PR TITLE
Change default list length in admin views from 10 to 20 items and fix editing of row count in pager

### PR DESCRIPTION
**Pull request by sturufous**

### CODE REVIEW

File: libs/npm/core/src/components/grid-table/Pager.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import React from 'react';
 import { BiFirstPage, BiLastPage } from 'react-icons/bi';
 
-import { Button, ButtonVariant } from '../button';
-import { Row } from '../flex';
-import { Text } from '../form/text';
-import { Show } from '../show';
+import { Button, ButtonVariant } from 'tno-core';
```
Comment (Fosol) [inline libs/npm/core/src/components/grid-table/Pager.tsx:4]: Cannot self reference

**Review by Fosol — CHANGES_REQUESTED** [review]

File: libs/npm/core/src/components/grid-table/Pager.tsx (hunk 1 above)
Comment (sturufous) [reply libs/npm/core/src/components/grid-table/Pager.tsx]: Those changes weren't supposed to be there. The commit now shows how I wanted it to look.

**Review by Fosol — COMMENTED** [review]
We still need a new version of the `tno-core` library to be published and referenced.  Looking forward to seeing this Monday.

**Review by Fosol — APPROVED** [review]

File: libs/npm/core/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "tno-core",
   "description": "TNO shared library",
-  "version": "0.0.187",
+  "version": "0.0.190",
```
Comment (Fosol) [inline libs/npm/core/package.json:4]: Does not match

**Review by Fosol — CHANGES_REQUESTED** [review]

**Comment by Fosol** [comment]
I've included this change in tno-core:0.0.193

**Pull request closed without merging**